vue-components/src/components/LookupMenu.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+	<div :class="[ 'wikit', 'wikit-LookupMenu' ]">
+		<div class="wikit-LookupMenu__item" :key="index" v-for="(menuItem, index) in menuItems">
+			<div class="wikit-LookupMenu__item__label">
+				{{ menuItem.label }}
+			</div>
+			<div class="wikit-LookupMenu__item__description">
+				{{ menuItem.description }}
+			</div>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+/**
+ * This is an internal component which used by the Lookup component.
+ */
+export default Vue.extend( {
+	name: 'LookupMenu',
+	props: {
+		menuItems: {
+			type: Array,
+			default: (): [] => [],
+		},
+	},
+	watch: {
+		menuItems(): void {
+			const menuItems = this.$el.querySelectorAll( '.wikit-LookupMenu__item' );
+			if ( menuItems.length > 6 ) {
```

Comment:
I see why 6 but it could be considered a magic number, is it possible to add a comment explaining it here (that the menu should be scrollable with more than 6 items long, I assume)?